Objects and Functions - 5.js

```diff
@@ -0,0 +1,23 @@
+/*
+-        Напишите функцию, которая определяет является ли число простым или нет
+ */
+
+let num = 115;
+
+function prime(a) {
```

Comment:
1) Bad naming for function. It is better to use something like `isPrime()`.
2) This function should return boolean values, not to to console.